src/mockApi/getData.js

```diff
@@ -0,0 +1,52 @@
+import { PAGES_KEYS, baseURL } from '../shared/constants';
+import { filterProgress, filterCurrentTasks } from '../shared/helpers';
+
+// import { getFirestore, collection, getDocs } from 'firebase/firestore';
+
+// async function getData(db) {
+//   const querySnapshot = await getDocs(collection(db, 'users'));
+//   querySnapshot.forEach((doc) => {
+//     // doc.data() is never undefined for query doc snapshots
+//     console.log(doc.data());
+//   });
+// }
+
+// getData(db);
+
+// export async function getUsers(app) {
+//   const db = getFirestore(app);
+//   const usersCol = collection(db, 'users');
+//   const usersSnapshot = await getDocs(usersCol);
+//   const usersList = usersSnapshot.docs.map((doc) => doc.data());
+//   console.log(usersList);
+// }
```

Comment:
Remove all commented out code. You can keep it locally if you need it.

Reply:
Ok, deleted.